sdk/synapse/Azure.Analytics.Synapse.Artifacts/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.0.0-preview.14 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.0.0-preview.14 (2022-03-08)
+- Add ScriptActivity
```

Comment:
nit: please change verb to past-tense and use a period at the end of the sentence.
```suggestion
- Added ScriptActivity.
```